Cargo.lock

```diff
@@ -214,6 +214,7 @@ dependencies = [
 name = "bootstrap"
 version = "0.0.0"
 dependencies = [
+ "bstr",
```

Comment:
In general more dependencies for bootstrap is something I'd like to work to avoid, at least until we have a ship-it-as-a-binary story in place. Can we maybe ditch bstr in favor of some manual similar logic? I think we're only using it to trim() a &[u8] which seems hopefully not too hard to hand-roll.